.rubocop_cocoapods.yml

```diff
@@ -94,6 +94,9 @@ EachWithObject:
 Style/SpecialGlobalVars:
   Enabled: false
 
+Metrics/LineLength:
```

Comment:
Do you want me to upstream these changes? Looks like [cocoapods/cocoapods](https://github.com/CocoaPods/CocoaPods/blob/master/.rubocop_todo.yml#L29) also has a different `Metric/LineLength` maxiumum set.